.github/workflows/jenkins-x/changelog.sh

```diff
@@ -13,9 +13,7 @@ echo "setting git user"
 git config --global user.name jenkins-x-bot-test
 git config --global user.email "jenkins-x@googlegroups.com"
 
-git add * || true
-git commit -a -m "chore: release $VERSION" --allow-empty
-git tag -fa v$VERSION -m "Release version $VERSION"
+git tag -fa v$VERSION -m "Release version $VERSION" HEAD
```

Comment:
coventional commit message format always.

Reply:
start it with `chore:`  ?